Tools/README

```diff
@@ -15,11 +25,16 @@ i18n            Tools for internationalization. pygettext.py
                 and msgfmt.py generates a binary message catalog
                 from a catalog in text format.
 
+importbench     A set of micro-benchmarks for various import scenarios.
+
 iobench         Benchmark for the new Python I/O system. (*)
 
 msi             Support for packaging Python as an MSI package on Windows.
 
-parser          Un-parsing tool to generate code from an AST.
+nuget           A package manager for Microsoft . NET applications.
+
+patchcheck      Tools for checking and applying patches to the Python source code
+                and verifying the integrity of patch files. 
```

Comment:
Please remove trailing spaces from here and the other additions:

```suggestion
                and verifying the integrity of patch files.
```

Reply:
Done